docs/surveys/salary-survey/2023.rst

```diff
@@ -0,0 +1,3948 @@
+   +--------------------------------------------------------+-----+-------+
+
+.. raw:: html
+
+   </div><div class="tab__content">
+
+.. table:: Current work search status - contractors
+   :widths: 70 15 15
+   :name: tbl-2023-current-work-search-status-contractors
+   :class: std3col
+
+   +--------------------------------------------------------+-----+-------+
+   | Search status                                          | No. | %     |
+   +========================================================+=====+=======+
+   | Not actively looking, but open to offers               | 30  | 38.0% |
+   +--------------------------------------------------------+-----+-------+
+   | Actively looking                                       | 23  | 29.1% |
+   +--------------------------------------------------------+-----+-------+
+   | Not actively looking, not open to offers               | 18  | 22.8% |
+   +--------------------------------------------------------+-----+-------+
+   | Not looking yet, but expect to be within the next year | 8   | 10.1% |
+   +--------------------------------------------------------+-----+-------+
+
+.. raw:: html
+
+	</div></div>
+
+Job security and stability
+--------------------------
+
+.. raw:: html
+   
+   <details><summary>What we asked</summary>
+
+.. container:: question
+
+   |icon-question| How would you characterize your current feelings of job security and stability, or contract/freelance income security and stability, compared to this time last year?
+
+   - I'm less confident in my job security or contract/freelance income security than I was last year
+   - My confidence level is around the same
+   - I'm more confident in my job security or contract/freelance income security than I was last year
+
+.. raw:: html
+
+   </details>
+
+Note: Employees and contractors who had indicated that they were currently unemployed were not shown this question. 
+
+Just over half of all respondents (54.1%) reported feeling the same level of job security and stability, compared to the previous year. 30.5% felt less secure, and only 15.4% felt more secure. The difference in responses between employees and contractors for this question was not significantly different.
+
+.. table:: Feelings of job security and stability
+   :widths: 70 15 15
+   :name: tbl-2023-feelings-of-job-security-combined
+   :class: std3col
+
+   +----------------------------------------+-----+-------+
+   | Feelings of job security and stability | No. | %     |
+   +========================================+=====+=======+
+   | The same as last year                  | 531 | 54.1% |
+   +----------------------------------------+-----+-------+
+   | Less secure than last year             | 299 | 30.5% |
+   +----------------------------------------+-----+-------+
+   | More secure than last year             | 151 | 15.4% |
+   +----------------------------------------+-----+-------+
+
+Job offer considerations
```

Comment:
I can't see the results summary or breakdown in the page, was this intentional due to sample size?

Reply:
Ah, this was in my notes on Slack: I haven't done the data processing for the "job offer considerations" section. I was thinking I could take it out of this report and we could publish that as a separate report later.

Just coming out of post-conference craziness, I am going to look at this again and see if the totals can be pulled out of the spreadsheet as is, or if I need to do additional processing on them.

Reply:
@thatdocslady so I've done a basic overview of this section but the main sticking point has been that I felt it needed a chart to really illustrate it, but because the "obvious" factors are so heavily weighted towards the positive, it looks kinda odd.  What do you think? https://writethedocs-www--2131.org.readthedocs.build/surveys/salary-survey/2023/#job-offer-considerations

Reply:
@goatlady actually the chart and table look great! It gives you an idea of where things are, and it's more about what's important rather than what's positive. If nothing else, it shows you that people care about a lot of things when they consider jobs. 

If you really don't like how this chart looks, maybe we can use a different style of graph like the graph in the section "Factors affecting employee salary satisfaction"? You can have all of the lines start at the same axis point but have each line contain the different colors. I'm probably not explaining this properly :)

Reply:
@thatdocslady thanks Mikey, I know what you mean... I've left it for now, concentrated in adding some other charts instead.